### PR TITLE
Rename to BLD and RUN in Treasury

### LINKED ISSUE #2786: Rename currencies in Treasury code
related: https://github.com/Agoric/dapp-token-economy/issues/143

## What is the Problem Being Solved?

Scones, Gov, etc

## Description of the Design

new names throughout.  Watch for conflicts with communication to the UI.

## Security Considerations

none

## Test Plan

rename example currencies, too?

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
Merging should be coordinated with dapp-token-economy.  https://github.com/Agoric/dapp-token-economy/pull/216

This changes Keywords in Treasury, and one place in bootstrap vat.

This changes variable names throughout the Treasury code.

closes #2786

### CODE REVIEW

File: packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js [hunk 1]
```diff
@@ -277,7 +277,7 @@ export function buildRootObject(vatPowers, vatParameters) {
             },
             want: {
               // We just throw away our governance tokens.
-              Governance: amountMath.make(0n, govBrand),
+              Bld: amountMath.make(0n, govBrand),
```
Comment (dtribble) [inline packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js]: Would be capitalized.  BUT!  Governance is actaully the bank liquidity token, and is NOT  BLD.

File: packages/treasury/src/liquidation.js [hunk 2]
```diff
@@ -30,50 +30,47 @@ export async function liquidate(
...
-// The default strategy converts of all the collateral to scones using autoswap,
-// and refunds any excess scones.
+// The default strategy converts of all the collateral to RUN using autoswap,
+// and refunds any excess RUN.
 export function makeDefaultLiquidationStrategy(autoswap) {
   function keywordMapping() {
     return harden({
       Collateral: 'In',
-      Scones: 'Out',
+      Run: 'Out',
```
Comment (dtribble) [inline packages/treasury/src/liquidation.js]: I think the keyword should be `RUN`. I'm not completely sure however.

File: packages/treasury/src/stablecoinMachine.js [hunk 3]
```diff
@@ -49,16 +49,13 @@ export async function start(zcf) {
     liquidationInstall,
   } = zcf.getTerms();
 
-  const [sconeMint, govMint] = await Promise.all([
-    zcf.makeZCFMint('Scones', undefined, harden({ decimalPlaces: 6 })),
-    zcf.makeZCFMint('Governance', undefined, harden({ decimalPlaces: 6 })),
```
Comment (dtribble) [inline packages/treasury/src/stablecoinMachine.js:54]: Governance is not BLD

**Review by dtribble — CHANGES_REQUESTED** [review]
I stopped midway because too many governance/BLD changes. Governance is not BLD.

File: packages/treasury/src/stablecoinMachine.js (hunk 3 above)
Comment (Chris-Hibbert) [reply packages/treasury/src/stablecoinMachine.js:54]: fixed.

File: packages/treasury/src/liquidation.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/treasury/src/liquidation.js]: I'm okay with that. It seemed counter to JS style, but nothing is complaining about it, so I'm happy to see if we like all-caps keywords.

**Review by dtribble — APPROVED** [review]

**Pull request merged**